src/views/ViewTrackDetails.tsx

```diff
@@ -26,12 +28,28 @@ export default function ViewTrackDetails() {
   const invalidate = useInvalidate();
 
   const [formData, setFormData] = useState<
-    Pick<Track, 'title' | 'artists' | 'album' | 'genres'>
+    Pick<
+      Track,
+      | 'title'
+      | 'artists'
+      | 'album'
+      | 'genres'
+      | 'year'
+      | 'duration'
+      | 'track'
+      | 'disk'
+      | 'path'
+    >
   >({
     title: track.title ?? '',
     artists: track.artists,
     album: track.album ?? '',
     genres: track.genres,
+    year: track.year ?? 0,
+    duration: track.duration,
+    track: track.track ?? { no: 0, of: 0 },
+    disk: track.disk ?? { no: 0, of: 0 },
```

Comment:
I would be careful with those defaults. The real default should probably be `{ no: null, of: null }`. Also, `track.track` and `track.disk` should technically always be defined.

Reply:
fixed to individual fields `track_no`, `track_of`, `disk_no`, `disk_of`...